### PR TITLE
Add connectOptions to doc block

### PR DESCRIPTION
**Pull request by TekkCraft**

### CODE REVIEW

**Review by ravage84 — APPROVED** [review]

File: lib/Cake/Routing/Router.php [hunk 1]
```diff
@@ -523,6 +523,7 @@ public static function namedConfig() {
  * - 'id' - The regular expression fragment to use when matching IDs. By default, matches
  *    integer values and UUIDs.
  * - 'prefix' - URL prefix to use for the generated routes. Defaults to '/'.
+ * - ‘connectOptions' – Custom options for connecting the routes.
```
Comment (dereuromark) [inline lib/Cake/Routing/Router.php]: wrong ` at the beginning.

Comment (TekkCraft) [inline lib/Cake/Routing/Router.php]: @dereuromark thanks, fixed.

**Comment by dereuromark** [comment]
Thank you!

**Comment by dereuromark** [comment]
Looks like it is also missing @ https://github.com/cakephp/cakephp/blob/master/src/Routing/RouteBuilder.php#L296

**Pull request merged**